packages/backend-for-frontend/src/api/notificationConfigApiConverter.ts

```diff
@@ -0,0 +1,18 @@
+import { bffApi, notificationConfigApi } from "pagopa-interop-api-clients";
+
+export function toBffApiTenantNotificationConfig(
+  tenantNotificationConfig: notificationConfigApi.TenantNotificationConfig
+): bffApi.TenantNotificationConfig {
+  return {
+    enabled: tenantNotificationConfig.enabled,
```

Comment:
the BFF responses omit config id, createdAt, updatedAt, hence the conversions here are needed